tasks/kubevirt-disk-uploader/0.5.0/kubevirt-disk-uploader.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     Automates the extraction of disk and uploads it to a container registry,
     to be used in multiple Kubernetes clusters.
   params:
-  - name: VM_NAME
+  - name: POD_NAME
```

Comment:
do not update older manifest, since it will not work with v0.5.0 image. Instead create new manifest

Reply:
Yep you're absolutely right, will do.